src/org/daisy/dotify/formatter/impl/row/SegmentProcessor.java

```diff
@@ -467,6 +471,9 @@ public boolean hasSignificantContent() {
             case Identifier:
                 applyAfterLeader((IdentifierSegment) s);
                 return Optional.empty();
+            case ExternalReference:
+                externalReference = ((ExternalReferenceSegment) s).getExternalReference();
```

Comment:
if `externalReference` has already a value (which can happen if you would have more than one external reference object in a block and which is perfectly legal by the OBFL definition) then an `UnsupportedOperationException` should be thrown, I think.

Reply:
Yes you are right that it might happen if you have two references on the same row but then they are still in the same block which means that they only have one composite object reference.